nsjail.cc

```diff
@@ -53,6 +57,9 @@ static void sigHandler(int sig) {
 	if (sig == SIGCHLD) {
 		return;
 	}
+	if (sig == SIGPIPE) {
```

Comment:
Could be `if (sig == SIGALRM || sig == SIGCHLD || sig == SIGPIPE)`

Reply:
I just kept the code structure used earlier.
However, with 3 instead of 2 repetitions this might be better :)